code/modules/jobs/job_types/captain.dm

```diff
@@ -54,6 +54,7 @@
 	duffelbag = /obj/item/storage/backpack/duffelbag/captain
 
 	implants = list(/obj/item/implant/mindshield)
+	cyberimps = list(/obj/item/organ/cyberimp/skillChip/security) //Technically not a member of security but fuck it why not, gives an option that isn't using lethals
```

Comment:
Keep it to security only, captain isnt a member of sec and shouldn't be making non lethal arrests.

Reply:
Fair enough